src/__tests__/httpYacApi.test.ts

```diff
@@ -712,51 +642,47 @@ GET  http://localhost:8080/test
 ###
 GET  http://localhost:8080/test?author={{slideshow.author}}
       `);
-
       expect(result).toBeTruthy();
       const requests = await mockedEndpoints.getSeenRequests();
       expect(requests[1].url).toBe('http://localhost:8080/test?author=httpyac');
     });
+
     it('set object variable with number', async () => {
       initFileProvider();
-      const mockedEndpoints = await localServer
-        .forGet('/get')
-        .thenReply(200, JSON.stringify({ foo: { test: 1 } }), { 'content-type': 'application/json' });
-
+      const mockedEndpoints = await localServer.forGet('/get')
+        .thenJson(200, { foo: { test: 1 } });
       const result = await exec(`
 # @name objectNumber
 GET http://localhost:8080/get
 @foo={{objectNumber.foo}}
+
 ###
 GET http://localhost:8080/get?test={{foo.test}}
-`);
+      `);
       expect(result).toBeTruthy();
       const requests = await mockedEndpoints.getSeenRequests();
       expect(requests[0].url).toBe('http://localhost:8080/get');
       expect(requests[1].url).toBe('http://localhost:8080/get?test=1');
     });
+
     it('direct replace variable', async () => {
       initFileProvider();
-      await localServer.forGet('/test').thenReply(200, JSON.stringify({ slideshow: { author: 'httpyac' } }), {
-        'content-type': 'application/json',
-      });
-      const mockedEndpoints = await localServer.forGet('/text').thenReply(200, 'foo', {
-        'content-type': 'text/plain',
-      });
-
+      await localServer.forGet('/test').thenJson(200, { slideshow: { author: 'httpyac' } });
+      const mockedEndpoints = await localServer.forGet('/text').thenJson(200, { slideshow: { author: 'foo' } });
```

Comment:
IMHO let's use a json response here. 'cause any other response may not be parsed and therefore never used.